src/Giraffe.Razor/Giraffe.Razor.fsproj

```diff
@@ -36,13 +36,14 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="2.2.*" />
-    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="2.2.*" />
-    <PackageReference Include="Microsoft.Extensions.FileProviders.Physical" Version="2.2.*" />
-    <PackageReference Include="Microsoft.Extensions.Primitives" Version="2.2.*" />
-    <PackageReference Include="Microsoft.AspNetCore.Antiforgery" Version="2.2.*" />
-    <PackageReference Include="TaskBuilder.fs" Version="2.1.*" />
-    <PackageReference Include="Giraffe" Version="3.6.*" />
+    <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="2.2.0" />
```

Comment:
I think we'd want to remove these and rely on `FrameworkReference` for `netcoreapp3.0` and up for some of these built-in and aspnetcore packages.